packages/block-editor/src/hooks/color.scss

```diff
@@ -0,0 +1,85 @@
+.color-block-support-panel {
+	.block-editor-contrast-checker {
+		/**
+		 * Contrast checkers are forced to the bottom of the panel so all
+		 * injected color controls can appear as a single item group without
+		 * the contrast checkers suddenly appearing between items.
+		 */
+		order: 9999;
+		grid-column: span 2;
+		margin-top: $grid-unit-20;
+
+		.components-notice__content {
+			margin-right: 0;
+		}
+	}
+
+	/* Increased specificity required to remove the slot wrapper's row gap */
+	&#{&} {
+		.color-block-support-panel__inner-wrapper {
+			row-gap: 0;
+		}
+	}
+
+	/**
+	 * The following styles replicate the separated border of the
+	 * `ItemGroup` component but allows for hidden items. This is because
+	 * to maintain the order of `ToolsPanel` controls, each `ToolsPanelItem`
+	 * must at least render a placeholder which would otherwise interfere
+	 * with the `:last-child` styles.
+	 */
+	.block-editor-tools-panel-color-gradient-settings__item {
```

Comment:
Should we move these styles to the component style itself (style.scss of the same folder of the component). Or are these styles specific to this particular usage of this component. If I'm not wrong this component is specific to hooks anyway.

Reply:
(Similar question for the remaining styles here)

Reply:
Currently, the component is only used in the color block supports hook but could be used to add a color control to any `ToolsPanel`. I can move these styles to live in the `color-gradients/style.scss` if that is better.